core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/pair/Pair.java

```diff
@@ -21,13 +21,15 @@
 import org.apache.commons.lang3.builder.HashCodeBuilder;
 import uk.gov.gchq.gaffer.commonutil.ToStringBuilder;
 
+import java.io.Serializable;
+
 /**
  * A simple class to contain a pair of items.
  *
  * @param <F> type of first item in the pair
  * @param <S> type of second item in the pair
  */
-public class Pair<F, S> {
+public class Pair<F, S> implements Serializable {
```

Comment:
Not sure this can be marked as `Serializable` as there's no guarantee that `F` and `S` are `Serializable`. Can you use a custom pair class?